apps/studio/pages/project/[ref]/settings/auth.tsx

```diff
@@ -1,53 +1,89 @@
-import { PermissionAction } from '@supabase/shared-types/out/constants'
-
-import {
-  AdvancedAuthSettingsForm,
-  BasicAuthSettingsForm,
-  SmtpForm,
-  ThirdPartyAuthForm,
-} from 'components/interfaces/Auth'
 import SettingsLayout from 'components/layouts/ProjectSettingsLayout/SettingsLayout'
-import {
-  ScaffoldContainer,
-  ScaffoldDescription,
-  ScaffoldHeader,
-  ScaffoldTitle,
-} from 'components/layouts/Scaffold'
-import NoPermission from 'components/ui/NoPermission'
-import { GenericSkeletonLoader } from 'components/ui/ShimmeringLoader'
-import { useCheckPermissions, usePermissionsLoaded } from 'hooks/misc/useCheckPermissions'
+import ProductEmptyState from 'components/to-be-cleaned/ProductEmptyState'
+import { useSelectedProject } from 'hooks/misc/useSelectedProject'
+import { ChevronRight } from 'lucide-react'
+import Link from 'next/link'
+import { useParams } from 'common'
 import type { NextPageWithLayout } from 'types'
 
-const AuthSettings: NextPageWithLayout = () => {
-  const isPermissionsLoaded = usePermissionsLoaded()
-  // TODO: check if these permissions cover third party auth as well
-  const canReadAuthSettings = useCheckPermissions(PermissionAction.READ, 'custom_config_gotrue')
+const ProjectSettings: NextPageWithLayout = () => {
+  const { ref } = useParams() as { ref: string }
 
   return (
-    <>
-      <ScaffoldContainer>
-        <ScaffoldHeader>
-          <ScaffoldTitle>Auth settings</ScaffoldTitle>
-          <ScaffoldDescription>Configure security and user session settings</ScaffoldDescription>
-        </ScaffoldHeader>
-      </ScaffoldContainer>
-      <ScaffoldContainer className="flex flex-col gap-10" bottomPadding>
-        {!isPermissionsLoaded ? (
-          <GenericSkeletonLoader />
-        ) : !canReadAuthSettings ? (
-          <NoPermission isFullPage resourceText="access your project's authentication settings" />
-        ) : (
-          <>
-            <BasicAuthSettingsForm />
-            <SmtpForm />
-            <AdvancedAuthSettingsForm />
-            <ThirdPartyAuthForm />
-          </>
-        )}
-      </ScaffoldContainer>
-    </>
+    <ProductEmptyState title="Authentication settings have moved">
```

Comment:
Noticed "Refresh tokens" is missing in this list - should we add it?
![image](https://github.com/user-attachments/assets/7b4ae44f-302e-499e-89e8-69f804557f8e)